internal/report.go

```diff
@@ -161,6 +163,13 @@ func buildTableReportBody(conv *Conv, srcTable string, issues map[string][]Schem
 				l = append(l, fmt.Sprintf("Column '%s' was added because this table didn't have a primary key. Spanner requires a primary key for every table", *syntheticPK))
 			}
 		}
+		if uniquePK != nil {
+			// Warning about using a column with unique constraint as primary key
+			// in case primary key is absent.
+			if p.severity == warning {
+				l = append(l, fmt.Sprintf("Column '%s' was used as primary key because it had UNIQUE constraint and this table didn't have a primary key. Spanner requires a primary key for every table", strings.Join(uniquePK, ", ")))
```

Comment:
We don't mention that we dropped the UNIQUE constraint. How about we rephrase this to
"UNIQUE constraint on column(s) '%s' replaced with primary key since this table didn't have one. Spanner requires a primary key for every table"

Reply:
rephrased as suggested